packages/xo-server-test/src/backupNg/backupNg.spec.js

```diff
@@ -66,66 +65,57 @@ const validateOperationTask = (task, props) => {
   })
 }
 
-// Note: `bypassVdiChainsCheck` must be enabled because the XAPI might be not
-// able to coalesce VDIs as fast as the tests run.
-//
-// See https://xen-orchestra.com/docs/backup_troubleshooting.html#vdi-chain-protection
 describe('backupNg', () => {
-  let defaultBackupNg
-
-  beforeAll(() => {
-    defaultBackupNg = {
-      name: 'default-backupNg',
-      mode: 'full',
-      vms: {
-        id: config.vms.default,
-      },
-      settings: {
-        '': {
-          reportWhen: 'never',
-        },
-      },
-    }
-  })
-
   describe('.createJob() :', () => {
     it('creates a new backup job without schedules', async () => {
-      const backupNg = await xo.createTempBackupNgJob(defaultBackupNg)
+      const jobInput = {
+        mode: 'full',
+        vms: {
+          id: config.vms.default,
+        },
+      }
+      const backupNg = await xo.createTempBackupNgJob(jobInput)
+
       expect(backupNg).toMatchSnapshot({
         id: expect.any(String),
+        name: expect.any(String),
         userId: expect.any(String),
         vms: expect.any(Object),
       })
-      expect(backupNg.vms).toEqual(defaultBackupNg.vms)
+      expect(backupNg.name).toBe(jobInput.name)
+      expect(backupNg.vms).toEqual(jobInput.vms)
       expect(backupNg.userId).toBe(xo._user.id)
```

Comment:
Can you mutualize this in a dedicated method?

Reply:
What do you think about not using snapshots in this cases?

Reply:
I don't care about snaphots, do what's better for this case.